boards/nrf51dk/Makefile-app

```diff
@@ -0,0 +1,12 @@
+# Makefile for loading applications for the nrf51dk
+
+NRF_LOAD = $(TOCK_USERLAND_BASE_DIR)/tools/program/nrf51dk.py
+
+# upload programs to nrf51dk
+.PHONY: program
+program: $(BUILDDIR)/app.bin
```

Comment:
`program` is meant to be for programming with a bootloader (like the stormloader on the Firestorm). Can you get rid of this and just keep `flash`. We'll need to unify this across the NRF Makefiles, but i'll handle that in a separate pull request